src/Raven.Studio/typescript/models/database/dbGroup/databaseGroupGraph.ts

```diff
@@ -687,23 +687,21 @@ class databaseGroupGraph {
         });
     }
     
-    static getUniqueTaskId(task: Raven.Client.Documents.Operations.OngoingTasks.OngoingTask): string {
-        if (task.TaskType === "PullReplicationAsHub") {
-            const hubTask = task as Raven.Client.Documents.Operations.OngoingTasks.OngoingTaskPullReplicationAsHub;
-            // since hub is generic definition we need to differentiate between instances
-            return hubTask.TaskId + ":" + hubTask.DestinationDatabase + "@" + hubTask.DestinationUrl;
```

Comment:
I wonder what was purpose of this code. Did you check that?

Reply:
This code was used to differentiate between the Sinks tasks 
(those that are actively connected to the Hub, i.e. children, not a sink task that is actually defined on a node..)
and to show all of them on the Graph

We don't need it anymore since we will not show the 'children sinks' on the graph now
As discussed - better to show the active sinks as children of the Hub item itself on the Graph 

Either way, this issue should be 'blocked' since we really need issue[ RavenDB-17125](https://issues.hibernatingrhinos.com/issue/RavenDB-17125) first
Because right now the node the Hub is attached to in the Graph is wrong...

Reply:
ok let's wait for RavenDB-17125 first.